### DISCUSSION TITLE
Workflow (Github Actions) not working after migrating the account from personal to free organization

### DISCUSSION THREAD

**Discussion by SquirrelDeveloper** [community/community#128376] [Actions]
### Select Topic Area

Bug

### Body

I have the following public repos https://github.com/SquirrelCorporation/SquirrelServersManager
And i have converted them from personal to organization.
Since then, the workflows are not executed anymore, either manually or automatically (see [here](https://github.com/SquirrelCorporation/SquirrelServersManager/actions/workflows/node.js.yml))
Can help me fix it, or is it a bug of the migration ?

**Comment by SquirrelDeveloper** [comment]
It seems I have fixed the problem, by creating a totally new, un related workflow, all the others seems to work again

**Reply by vixducis** [reply]
This worked for me as well.

**Reply by TrungNguyen78** [reply]
thanks, this worked for me also.

**Reply by lateefojulari** [reply]
Worked for me too. Thanks.

**Reply by nosa-stonetoned** [reply]
Thanks, a rename worked for me.

**Reply by ayodejii** [reply]
a rename worked for me too.

**Reply by seanvilya** [reply]
I had this same issue after migrating a GitHub Enterprise (cloud, not on-prem) from personal accounts to managed users. Had to create a PR (didn't even need to merge it) with some random new workflow. Was required for every repo, so I had to write a script to apply it to the whole org.
